db/migrate/20190410130528_add_delivery_stats_to_endpoint.rb

```diff
@@ -0,0 +1,7 @@
+class AddDeliveryStatsToEndpoint < ActiveRecord::Migration[5.2]
+  def change
+    add_column :endpoints, :last_delivery_status, :string
```

Comment:
Do we expect more than 3 states? If not, we could probably get away with making this column boolean+NULL

Reply:
Somehow string status feels better. Maybe later we will add more information about the delivery there.
If you insist, I'll make it boolean.

Reply:
No problem with that, just wanted to know what was the intent